src/docfx/build/page/BuildPage.cs

```diff
@@ -53,6 +52,24 @@ internal static class BuildPage
             };
 
             return (errors, output, publishItem);
+
+            void ResolveBreadcrumb()
+            {
+                if (!string.IsNullOrEmpty(metadata.BreadcrumbPath))
+                {
+                    var (breadcrumbError, breadcrumbPath, _) = context.DependencyResolver.ResolveLink(metadata.BreadcrumbPath, file, file, buildChild);
+                    errors.AddIfNotNull(breadcrumbError);
+                    metadata.BreadcrumbPath = breadcrumbPath;
+                }
+            }
+
+            async Task ResolveContributors()
+            {
+                List<Error> contributorErrors;
+                (contributorErrors, model.Author, model.Contributors, model.UpdatedAt) = await context.ContributionProvider.GetAuthorAndContributors(file, metadata.Author);
```

Comment:
Assign result to `model` outside local function? Otherwise it is hard to see where a `model` filed is changed unless read through all the local functions.

Reply:
see